bin/annex_sync

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+
+def psql(sql):
+    args = ['psql', '-U', 'vimc', '-d', 'montagu', '-c', sql]
+    subprocess.run(args, check = True)
+
+# It may be worthwhile setting the option 'synchronous_commit' to
+# 'on' by adding the line
+#
+#   WITH (synchronous_commit = on)
+#
+# at the end of the CREATE SUBSCRIPTION block here.  We can also
+# change this later with ALTER SUBSCRIPTION.
+#
+# https://www.postgresql.org/docs/10/static/sql-createsubscription.html
+# https://www.postgresql.org/docs/10/static/runtime-config-wal.html#guc-synchronous-commit
+def subscribe_start(d):
+    # This can't be done in a transaction, which given what it does is
+    # far from ideal!  If anything fails through this *will* leave the
+    # db in an unfortunately state.
```

Comment:
This feels like something we need to come up with a fix for.

Reply:
Ah, this _whole file_ should not be here - it should have been purged in ecab752

Reply:
removed now